contract-sdk/src/abi/env.rs

```diff
@@ -28,4 +38,18 @@ impl Env for HostEnv {
     fn query<Q: Into<QueryRequest>>(&self, q: Q) -> QueryResponse {
         query(q.into())
     }
+
+    fn address_for_instance(&self, instance_id: InstanceId) -> Address {
+        // Prepare a region for response.
+        let dst = vec![0; 21];
```

Comment:
Any reason for using vec instead of an array?

Reply:
no reason, fixed :-)